website/docs/project-templates.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "Project templates"
+---
+
+We prepared some useful templates for you to boost your start with the Wave app development. Just use `wave init` command and pick the one you like.
+
+![template-wave-init](assets/template-wave-init.gif)
```

Comment:
Let's reuse the existing gif - https://wave.h2o.ai/blog/release-0.24/#wave-init-cli.